### ISSUE TITLE
activate it just on paste?

### ISSUE THREAD

**Issue by jacquescrocker** [chfritz/atom-sane-indentation#6]
When I enable `atom-sane-indentation`, it works great for when pasting in blocks of code (it actually goes to the right place).

But for when I'm editing the code myself, it seems to throw away all my editor configurations and tab spacing ends up completely off.

Is there a way to enable this plugin for just pasting?

**Comment by chfritz** [comment]
This shouldn't happen. Which language are you experiencing this with?

I don't know the answer to the question you ask.

**Comment by jacquescrocker** [comment]
Yeah, sorry my post wasnt too clear. I'll try to get you more details.
